MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs

```diff
@@ -388,6 +388,35 @@ public void UpdateServerTimeOffset(long currentServerTime)
 			AnimationManager.UpdateServerTimeOffset(currentServerTime);
 		}
 
+		public void DeclarePreallocatedActors(GameObject[] objects, string guidSeed)
```

Comment:
When is it not safe to execute this? We should protect against calling it too early by pending the operation.

Reply:
I need to check. I called this during execution (onStarted) during testing, but maybe the sync phase would be better if it's supported.